configs/webpack.client.config.js

```diff
@@ -219,6 +219,17 @@ module.exports = (env, argv) => {
         filename: "[name].[chunkhash].css",
       }),
       new BuildManifestPlugin({ buildId }),
+      new webpack.DefinePlugin({
+        ...Object.keys(process.env).reduce(
```

Comment:
Could we refactor this to `filter` by `startsWith` and then `reduce`?

Reply:
Sure, I can do that. I have a question about this. How would that help? 🙈  Performance?

Reply:
Just cleaner code but not a blocker!